backend-test/Controllers/CourseControllerTests.hs

```diff
@@ -122,6 +122,92 @@ runIndexTest label courses expected =
 runIndexTests :: [Test]
 runIndexTests = map (\(label, courses, expected) -> runIndexTest label courses expected) indexTestCases
 
+-- | Helper function to insert full (predefined) courses into the database
```

Comment:
This really doesn't need to be its own helper function. You can call `mapM_ insert_` directly.